packages/ui/src/components/CategoryLink/CategoryLink.tsx

```diff
@@ -0,0 +1,23 @@
+import type { Category, Culture } from '@prezly/sdk';
+import { getCategoryUrl, getLocalizedCategoryData, LocaleObject } from '@prezly/theme-kit-core';
+import Link from 'next/link';
+import { twMerge } from 'tailwind-merge';
+
+interface Props {
+    category: Category;
```

Comment:
It is recommended to `Pick<...>` only the properties the component needs to make it possible using the library with a wider range of `@prezly/sdk` package versions.

Read more about it here: https://github.com/prezly/theme-kit-js#prezly-sdk-dependency

Reply:
Updated https://github.com/prezly/theme-kit-js/pull/414/commits/7f5e4071f2ebd39b7cae32883fec555728b1ac8b